### DISCUSSION TITLE
Installed but doesnt show up

### DISCUSSION THREAD

**Discussion by Soup2305** [scfmod/FS22_TerraFarm#90] [Q&A]
I installed the latest version and it comes up in my Mod Selection in game and I turn it on, however when I'm in equipment none of the TerraFarm triggers to launch the menu or anything work.

**Comment by CavemanTechandGamming** [comment]
You need to make sure you're actually using something that is set up to work with terrafarm. If you open up the terrafarm zip file and look for XML configs folder it contains all the equipment config files for what will and won't work with terrafarm. If you open up any of those files you'll see the name of the mod in that file. And if there is one that you want to add to it there's a video on YouTube of how to add support for a particular mod to terrafarm.
